src/constants.js

```diff
@@ -2,4 +2,12 @@ export const NAV_LINKS = [
   'Popular', 'Now playing', 'Upcoming', 'Most rated',
 ];
 
-export const ENTER_KEY = 13;
+const PORT = 3000;
+
+export const API_URL = `http://localhost:${PORT}`;
+
+export const LIST_PAGE = 1;
```

Comment:
Name this constant more specific. It uses only for `Search`

Reply:
May it be```SEARCH_LIST_PAGE```?